data/jakarta-persistence/codegen/src/main/java/io/helidon/data/jakarta/persistence/codegen/JpqlKeywords.java

```diff
@@ -0,0 +1,229 @@
+/*
+ * Copyright (c) 2025 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.data.jakarta.persistence.codegen;
+
+/**
+ * JPQL keywords used to build query {@link String}.
```

Comment:
When JPQL changes in subsequent revisions of the specification, will we have to update this class?

Reply:
If JPQL changes, we'll have to update much more than just this class. :)